app/views/devise/registrations/edit.html.haml

```diff
@@ -11,10 +11,6 @@
   .field
     %h4= f.label :password_confirmation
     = f.password_field :password_confirmation, class: 'form-control', autocomplete: 'off'
-  .field
```

Comment:
Be careful: we don't want the current_user's password to be display when changing regular info of the user (ie: his CIS).
But, in case the user wants to update his password, we have to ask him to type his current password whatever.
That's a security.

Is this the actual behaviour?